test/integration/server0_p_10_and_later_test.exs

```diff
@@ -19,11 +20,14 @@ defmodule KafkaEx.Server0P10P1AndLater.Test do
     resp = create_topic(name, config)
     assert {:topic_already_exists, name} == parse_create_topic_resp(resp)
 
-    wait_for(fn ->
-      Enum.member?(existing_topics(), name)
-    end)
+    Enum.member?(existing_topics(), name)
```

Comment:
here, we don't wait anymore since the metadata will be refreshed immediatly

Reply:
This actually makes a lot more sense - this could be a use case in production code, and it would be annoying to make users insert a wait for this to be updated.